deployment/eclipse-kuksa/utils/hono-influxdb-connector/deploy_connector.sh

```diff
@@ -33,16 +33,14 @@ DOCKER_REGISTRY_SECRET=${9:-""}
 DOCKER_REGISTRY_SERVER=${10:-localhost}
 DOCKER_REGISTRY_USERNAME=${11:-""}
 DOCKER_REGISTRY_PASSWORD=${12:-pw}
-DOCKER_REGISTRY_EMAIL=${13:-user@domain.com}
-VERSION=${10,-"0.1.0"}
 
 NAMESPACE=hono
 DOCKER_IMAGE_NAME=hono-influxdb-connector
 
-VERSION="$(gradle properties -q | grep 'version:' | awk '{print $2}')"
+VERSION="0.1.0"
```

Comment:
we just release version 0.2.0, so we should adapt this here accordingly.

Reply:
Oh, yes, thanks for the hint! It's fixed in the latest commit.